### DISCUSSION TITLE
Skipping "installation with specific user creds" prompt

### DISCUSSION THREAD

**Discussion by kawishbit** [ChrisTitusTech/winutil#2057] [Q&A] [answered]
Hi everyone, It's my first time using winutil, I have to say that I'm loving it so far. However, I have a question, is there a way to skip this prompt below (i.e an option to indicate before install that I want to answer to every time).

`$userChoice = [System.Windows.MessageBox]::Show("Do you want to attempt $($Program.winget) installation with specific user credentials? Select 'Yes' to proceed or 'No' to skip.", "User Credential Prompt", [System.Windows.MessageBoxButton]::YesNo)`

I personally need this because sometimes I just want to leave the programs to install themselves while I go out, so I won't be able to click the prompt which will result to the other programs in the queue not being installed.

**Comment by kawishbit** [comment]
PS: If we don't have the feature at the moment, I'm happy to implement the feature and create a PR

**Comment by Marterich** [comment, marked as answer]
At the moment there's no option to disable the prompt.
However, I think the install logic is due for a rework anyways in the near future because the script got quite bloated, unreadable and not even close to all winget exit codes are implemented and sometimes the winutil therefore thinks the install has completely failed even though there was some kind of other meaning behind the exit codes, so if your motivated and up to the task, I would say go for it and code a PR.

Background: the logic with the prompt exists, because some programs (like Spotify) really want to be installed in the user context without admin rights and this was the first workaround Chris found but there are most definitely more elegant ways to accomplish the goal

**Comment by SyedArbaazHussain** [comment]
I am also getting the prompt for  "installation with specific user" on some of the applications I want to install.

I don't know how to pass this screen.
No matter what I have tried I cannot pass this prompt for user credentials.

It would be great if someone would help me by guiding me through to follow some steps to pass it.

**Reply by RichardPinka** [reply]
Yes. Same problem here :(  what is this about, I actually am not sure what "use my credentials" is and how to find them ..  my normal password and windows profile username does not work it seems to me.

**Reply by Nvaeplay** [reply]
Same, I guess i don't know what user credentials are, I also thought it was my users login info.
